app/build.gradle

```diff
@@ -25,6 +25,9 @@ android {
         versionCode versions.code // Notice that you may want to use BUILD_NUMBER from CI in real project with own CI.
         versionName versions.name
 
+        // Make generated apks contain version name and code
```

Comment:
nit: `.` at the end http://nedbatchelder.com/blog/201401/comments_should_be_sentences.html (learned that from PRs to Square projects)